extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -244,6 +249,9 @@ export class AmpAdApiHandler {
    * @private
    */
   sendEmbedInfo_(inViewport) {
+    if (!this.embedStateApi_) {
```

Comment:
`viewportCallback()` can be called anytime after element build. `this.embedStateApi_` may not be initiated when this function is called. And the check here.